### ISSUE TITLE
4.1 — Implement Custom CNN Architecture

### ISSUE THREAD

**Issue by A01799073** [A01799073/AguacAIte-Labs#14]
Develop a lightweight CNN model tailored for binary classification of chest X-rays.

- [x] Add conv layers + ReLU + batch norm
- [x] Add dropout
- [ ] Add final sigmoid layer

**Comment by A01799073** [comment]
Final sigmoid is intentionally omitted in the model.
The network outputs logits to be used with BCEWithLogitsLoss during training
